src/Checkin.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace Honeybadger;
+
+use Honeybadger\Exceptions\ServiceException;
+
+/**
+ * @see https://docs.honeybadger.io/api/check-ins/#create-a-check-in
+ */
+class Checkin
+{
+    /**
+     * Checkin identifier.
+     *
+     * @var string|null
+     */
+    public $id;
+
+    /**
+     * Checkin name.
+     *
+     * @var string|null
+     */
+    public $name;
+
+    /**
+     * Valid values are "simple" or "cron".
+     * If you specify "cron", then the "cron_schedule" field is required.
+     *
+     * @var string|null
+     */
+    public $scheduleType;
+
+    /**
+     * For simple check-ins, the amount of time that can elapse before the check-in is reported as missing.
+     * E.g., "1 day" would require a hit to the API daily to maintain the "reporting" status.
+     *
+     * @var string|null
+     */
+    public $reportPeriod;
+
+    /**
+     * The amount of time to allow a job to not report before it's reported as missing.
+     *
+     * @var string|null
+     */
+    public $gracePeriod;
+
+    /**
+     * For a scheduleType of "cron", the cron-compatible string that defines when the job should be expected to hit the API.
+     *
+     * @var string|null
+     */
+    public $cronSchedule;
+
+    /**
+     * The timezone setting for your server that is running the cron job to be monitored.
```

Comment:
I'm not sure what the format for the timezone looks like.. is it formatted like "GMT-4"?

Reply:
@joshuap @stympy Same as above.

Reply:
I have updated [the docs](https://docs.honeybadger.io/api/check-ins/#create-a-check-in) to include this info